docs/УправлениеИнтеграциями.md

```
@@ -0,0 +1,276 @@
+
+## Связанные подсистемы
+
+1. [Предопределенные значения](docs/ПредопределенныеЗначения.md)
+2. [Переопределения методов БСП](docs/ПереопределениеМетодовБСП.md)
+3. [Пользовательские функции](docs/ПользовательскиеФункции.md)
+
+## Порядок взаимодействия
+
+Для создания настроек интеграции необходимо создать предопределенные элементы справочников **пбп_ИнтегрируемыеСистемы**, **пбп_НастройкиИнтеграции**, **пбп_ИнтеграционныеПотоки** с помощью подсистемы предопределенных значений (см. [Предопределенные значения](docs/ПредопределенныеЗначения.md)). Рассмотрим заполнение данных на нескольких примерах:
+
+#### Заполнение настроек для обмена через FTP-Сервер
```

Markdownlint

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)